ethcore/src/transaction.rs

```diff
@@ -125,8 +125,8 @@ impl Transaction {
 			r: r,
 			s: s,
 			v: v + 27,
-			hash: RefCell::new(None),
-			sender: RefCell::new(None)
+			hash: RwLock::new(None),
```

Comment:
is `Transaction` really so substantial that references to it need to be thread-safe?